src/Todo.js

```diff
@@ -0,0 +1,80 @@
+class Todo {
+  static todoArray =[]
+
+  static storageUpdater;
+
+  static setUpdater(value) {
+    Todo.storageUpdater = value;
+  }
+
+  constructor(task) {
+    this.task = task;
+    this.isCompleted = false;
+    this.index = null;
+
+    Todo.todoArray.push(this);
+    Todo.recalculateIndex();
+
+    if (typeof Todo.storageUpdater === 'function') {
+      Todo.storageUpdater();
+    }
+  }
+
+  deleteTodo() {
+    const i = Todo.todoArray.indexOf(this);
+    Todo.todoArray.splice(i, 1);
+    Todo.recalculateIndex();
+    if (typeof Todo.storageUpdater === 'function') {
+      Todo.storageUpdater();
+    }
+  }
+
+  updateTodo(newTask) {
+    const i = Todo.todoArray.indexOf(this);
+    const todo = Todo.todoArray[i];
+    todo.task = newTask;
+  }
```

Comment:
- [ ] Great job implementing all the necessary functions, however when after I edit it doesn't store the data on local storage, kindly fix that.